.gitignore

```diff
@@ -35,3 +35,4 @@ cardigan/
 /server/config/js-assets.json
 
 /server/views/partials/critical.css.njk
+/client/scss/utilities/compiled_variables/
```

Comment:
Yes – what would a good strategy be to enforce a successful build step here? Or are there others?